src/Native/Unix/System.Native/pal_networking.c

```diff
@@ -782,6 +788,8 @@ static int32_t GetIPv4PacketInformation(struct cmsghdr* controlMessage, struct I
     ConvertInAddrToByteArray(&packetInfo->Address.Address[0], NUM_BYTES_IN_IPV4_ADDRESS, &pktinfo->ipi_addr);
 #if HAVE_IN_PKTINFO
     packetInfo->InterfaceIndex = (int32_t)pktinfo->ipi_ifindex;
+#elif defined(_AIX)
+    // TODO: SIOCGIFCONF based fallback
```

Comment:
I'd prefer it if we avoid adding these TODOs.  Since this is incomplete anyway, let's just leave them out and the appropriate fallbacks can be added when implementation is available for them.